crates/sui-framework/sources/math.move

```diff
@@ -64,4 +64,16 @@ module sui::math {
 
         (res as u64)
     }
+    
+    /// Return the value of a base raised to a power
+    public fun pow(base: u64, exponent: u64): u64 {    
+        let res = 1;
+        let i = 0;
+        while (i < exponent) {
+            res = (res * base);
+            i = i + 1 ;
+        };
+
+        (res as u64)
```

Comment:
This cast should be unnecessary (`sqrt` uses one because `res` was initially a `u128`).